lib/test_helper.js

```diff
@@ -1,5 +1,6 @@
 const fs = require( "fs" );
```

Comment:
Could you add /* eslint-disable no-console */ to the top of this file to suppress the eslint warnings about the use of console.log in this file?